release.json

```diff
@@ -18,7 +18,7 @@
     },
     "nightly-a7": {
         "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
+        "OMNIBUS_SOFTWARE_VERSION": "florentclarret/bump-gdbm",
```

Comment:
Is this change expected?

Reply:
Marking this PR to draft since this is not ready yet. We modified this to be able to use [this updated recipe](https://github.com/DataDog/omnibus-software/pull/439) that is not merged yet to build the docker image in this CI for testing purpose

Reply:
Fixed! We will not need gdbm anymore.